go/coinstacks/osmosis/api/handlers.go

```diff
@@ -88,39 +92,54 @@ func (h *Handler) StartWebsocket() error {
 	return nil
 }
 
+// TODO add osmosis apr
 func (h *Handler) GetInfo() (api.Info, error) {
 	info := Info{
 		BaseInfo: api.BaseInfo{
 			Network: "mainnet",
 		},
+		TotalSupply:      "0",
+		BondedTokens:     "0",
+		AnnualProvisions: "0",
+		CommunityTax:     "0",
```

Comment:
We can remove these values and from the type until we officially add proper apr calculations. Leaving `APR` since that is a required value to be passed into `getAccountData`